vital/bindings/python/vital/algo/algorithm_factory.cxx

```diff
@@ -45,7 +45,7 @@
 
 namespace py = pybind11;
 
-static void add_algorithm( const std::string& impl_name, std::string const& description, 
+static void add_algorithm( const std::string& impl_name, std::string const& description,
```

Comment:
Very minor nit especially considering existing code, but the mismatched placement of const & and type in the argument list is hard to read.

Probably for a different PR too, but it looks like the rest of this file suffers the same from function to function.